switcheroo/data_stores.py

```diff
@@ -0,0 +1,161 @@
+"Data stores that specifies where a Server stores its keys"
+from abc import ABC, abstractmethod
+from pathlib import Path
+from tempfile import TemporaryDirectory
+import boto3
+from switcheroo import paths
+from switcheroo.custom_keygen import KeyGen, KeyMetadata
+
+
+class DataStore(ABC):
+    "A server uses a DataStore to get public keys from somewhere."
+
+    def __init__(self, ssh_home: Path | None = None, temp: bool = False):
+        # If temp is true, will reset the home_dir to a temp file upon usage as a context manager
+        self.temp = temp
```

Comment:
private member should start with a leading _

Reply:
👍